### DISCUSSION TITLE
Back to differnt tenant

### DISCUSSION THREAD

**Discussion by Jens1986-11** [almenscorner/IntuneCD#182] [Q&A]
Hi all

i have following question.
is it possible to set a backup from one tenant to an other? 
i've tried the update function in a pipeline and it runs succesfully but nothing is restored on the new tenant.
the update step only takes one second.

![image](https://github.com/almenscorner/IntuneCD/assets/77617554/1603d3a3-0afd-40b1-b4cb-0b68a1078940)

any help is welcome

Thanks in advance

**Comment by almenscorner** [comment]
How is your pipeline setup? It looks like it cannot find the exported configs

**Reply by Jens1986-11** [reply]
Hi 

below is the pipeline for the restore.

I have already tried several things in the update command. the path with .\ before and without. but always with the same result

![image](https://github.com/almenscorner/IntuneCD/assets/77617554/6344b598-bb85-401c-8261-60aeb54c06ac)

all variables are entered and work when taking the backup.
below the variable of the REPO_DIR

![image](https://github.com/almenscorner/IntuneCD/assets/77617554/55c6c7f7-0b6e-4212-bea7-d83bf79f30e1)

Thanks

**Reply by almenscorner** [reply]
Try with the full path: `-p $(Build.SourcesDirectory)/Demo Backup` 

Also remove the repo dir env var as you are providing the path

**Reply by Jens1986-11** [reply]
Hi 
thanks for the push in the right direction.
I had added a step to look at the dir in the pipeline.
![image](https://github.com/almenscorner/IntuneCD/assets/77617554/fc558953-ffed-4d84-9194-d36488e75b3f)

After adjusting Demo backup to Prod-backup, the update will start and complete the Update.
![image](https://github.com/almenscorner/IntuneCD/assets/77617554/3c9c20ec-69cb-499c-9875-164fed04e88c)


Thanks for the help
